include/deal.II/dofs/dof_renumbering.h

```diff
@@ -1214,6 +1214,54 @@ namespace DoFRenumbering
    * @}
    */
 
+  /**
+   * @name Numberings based on properties of the finite element space
+   * @{
+   */
+
+  /**
+   * Sort DoFs by component and then by location of their support points, so
+   * that DoFs with the same support point are listed consecutively in the
+   * component order.
```

Comment:
I think if I read the second part of the sentence right, then you've got it exactly the wrong way around in the first part. Correct?

Reply:
If we sort by component first and then stably sort by support point location then we will get the desired ordering. I think the same is achieved by sorting by support point first and then (stably) sorting by component as well. I will figure out an unambiguous way to describe this.

Reply:
Ah, I see what you mean. I had taken the "first ... and then..." as major/minor sorting criteria where the latter is a tie-breaker if the former is the same. Maybe use "In a first step, do X. Following this, do Y in a second step." to make it clear that you were thinking of a temporal order.